### DISCUSSION TITLE
How to suppress/remove local variables in the completions list

### DISCUSSION THREAD

**Discussion by rkpatel33** [securingsincity/react-ace#532] [Q&A]
# Problem

I would like to suppress local variables/symbols from being suggested in the autocomplete list. I found this issue related detailing how to do it when using Ace directly, but could not figure out how to set completions using `react-ace`.

https://stackoverflow.com/questions/26991288/ace-editor-autocompletion-remove-local-variables/28500398

Any help appreciated. Thx.

**Comment by securingsincity** [comment]
You can do what they have in that answer using the example here https://github.com/securingsincity/react-ace/blob/master/docs/FAQ.md#how-do-i-add-language-snippets And add your completers by modifying the result of the import of the `language_tools`

**Comment by freddy-dev** [comment]
This worked for me!

```
import 'brace/ext/language_tools'

componentDidMount() {
    let langTools = ace.acequire('ace/ext/language_tools')
    langTools.setCompleters([langTools.snippetCompleter, langTools.keyWordCompleter])
}
```
